selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/webdriver/SeleniumWebDriverHelper.java

```diff
@@ -953,6 +954,31 @@ public boolean isVisible(WebElement webElement) {
     }
   }
 
+  /**
+   * Determines whether or not this element {@link WebElement} is selected (uses for checkboxes,
```

Comment:
Maybe better next:
Determines whether or not this element {@link WebElement} is selected.
<p>Note! Uses only for checkboxes and radiobuttons.

Reply:
and for all occurrence